vcf.c

```diff
@@ -1731,7 +1731,7 @@ int vcf_hdr_write(htsFile *fp, const bcf_hdr_t *h)
 
 void bcf_enc_vint(kstring_t *s, int n, int32_t *a, int wsize)
 {
-    int32_t max = INT32_MIN + 1, min = INT32_MAX;
+    int32_t max = BCF_MIN_BT_INT32 + 1, min = BCF_MAX_BT_INT32;
```

Comment:
These are boundaries on the **unencoded** `int32_t` values, so should just be

    int32_t max = INT32_MIN, min = INT32_MAX;

and on the lines below you need to lose the `+ 8`.

Reply:
Thanks for spotting that. I was under the impression they were match for match.

Reply:
Thanks — looks good to me now.